### DISCUSSION TITLE
Has anyone been able get back into their accounts?

### DISCUSSION THREAD

**Discussion by barisdx** [Discord-OTP-Forcer/Discord-OTP-Forcer#75] [General]
Hello. Firstly I'd like to thank the developer for giving us an opportunity to get back into our accounts. Although it is obvious that the chances of succeeding are very low (for 6 digit ones, approx. 4 in a million since there is an average 7,5 seconds delay for each attempt and for 8 digit ones around ten in 2.82 trillion which in worst case scenario takes 62619 years) the chances and still not 0. But at the same time the odds don't seem to be very gladdening. So, has anyone been able to get back into their accounts?

TLDR: The odds are very very low for both 6 and 8 digit codes, has anyone been able to get their accounts back?

**Comment by Chimmy2302** [comment]
I was wondering the same thing haha. I just installed the program yesterday, and I just wish to know as well if there is anyone else who succeeded in attaining their account back.
